sdk/src/simple_state/addresser.rs

```diff
@@ -167,6 +167,179 @@ fn calculate_hash_lengths(
 mod tests {
     use super::*;
 
+    #[test]
```

Comment:
Where these tests supposed to go into this commit?

Reply:
Yes, @peterschwarz suggested adding these tests for this function to the commit it was added so it would be simpler to see that the comment was actually addressed.